components/popover/test/index.js

```diff
@@ -30,22 +32,26 @@ describe( 'Popover', () => {
 		} );
 
 		it( 'should add window events', () => {
-			wrapper = mount( <Popover /> );
+			wrapper = TestUtils.renderIntoDocument( <Popover /> );
 			expect( Popover.prototype.toggleWindowEvents ).toHaveBeenCalledWith( true );
 			expect( Popover.prototype.computePopoverPosition ).toHaveBeenCalled();
 		} );
 
 		it( 'should remove window events', () => {
-			wrapper = mount( <Popover /> );
-			wrapper.unmount();
+			wrapper = TestUtils.renderIntoDocument( <Popover /> );
+			/* eslint-disable react/no-find-dom-node */
+			ReactDOM.unmountComponentAtNode( ReactDOM.findDOMNode( wrapper ).parentNode );
+			/* eslint-enable react/no-find-dom-node */
 
 			expect( Popover.prototype.toggleWindowEvents ).toHaveBeenCalledWith( false );
 		} );
 
 		it( 'should set offset and forced positions on changed position', () => {
-			wrapper = mount( <Popover /> );
+			const node = document.createElement( 'div' );
+			wrapper = ReactDOM.render( <Popover />, node );
```

Comment:
I had to use `ReactDOM.render` directly here because it was the only way to update (that I could see) the existing component.

Reply:
Should we remove the test to avoid using `render`?

Reply:
Well it works for preserving this specific test  It's possible there's an alternative way to test this behaviour without using `React.render` to update the existing component but I didn't come up with something in my sprint.  Personally, I'm fine with leaving as is for now.